<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="tomviz::CentralWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecentlyOpened">
     <property name="title">
      <string>R&amp;ecently Opened</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuRecentlyOpened"/>
    <addaction name="separator"/>
    <addaction name="actionSaveData"/>
    <addaction name="separator"/>
    <addaction name="actionSaveScreenshot"/>
    <addaction name="actionSaveMovie"/>
    <addaction name="actionSaveWeb"/>
    <addaction name="separator"/>
    <addaction name="actionLoadState"/>
    <addaction name="actionSaveState"/>
    <addaction name="separator"/>
    <addaction name="actionSaveDebuggingState"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuModules">
    <property name="title">
     <string>Vis&amp;ualization</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionMacros"/>
    <addaction name="actionPassiveAcquisition"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>&amp;Data Transforms</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTomography">
    <property name="title">
     <string>&amp;Tomography</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSegmentation">
    <property name="title">
     <string>Segmentation</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuData"/>
   <addaction name="menuSegmentation"/>
   <addaction name="menuTomography"/>
   <addaction name="menuModules"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>300</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Pipelines</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="pipelineBrowserWidget">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="tomviz::PipelineView" name="treeWidget">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="modulesToolbar">
   <property name="windowTitle">
    <string>Visualization Modules Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="pqAxesToolbar" name="axesToolbar">
   <property name="windowTitle">
    <string>Axes Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="utilitiesToolbar">
   <property name="windowTitle">
    <string>Utilities Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="pqCameraToolbar" name="cameraToolbar">
   <property name="windowTitle">
    <string>Camera Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidgetAnimation">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Animation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="pqPVAnimationWidget" name="animationWidget"/>
  </widget>
  <widget class="pqVCRToolbar" name="vcrToolbar">
   <property name="windowTitle">
    <string>VCR Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget_5">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>300</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QStackedWidget" name="propertiesPanelStackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="empty"/>
         <widget class="QScrollArea" name="dataPropertiesScrollArea">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="tomviz::DataPropertiesPanel" name="dataPropertiesPanel">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>30</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QScrollArea" name="modulePropertiesScrollArea">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="tomviz::ModulePropertiesPanel" name="modulePropertiesPanel">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>30</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QScrollArea" name="operatorPropertiesScrollArea">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="tomviz::OperatorPropertiesPanel" name="operatorPropertiesPanel">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>30</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetMessages">
   <property name="windowTitle">
    <string>Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="pqOutputWidget" name="outputWidget"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetPythonConsole">
   <property name="windowTitle">
    <string>Python Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="pqPythonShell" name="pythonConsole"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetLightsInspector">
   <property name="windowTitle">
    <string>Lights Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqLightsInspector" name="lightsInspector"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveDebuggingState">
   <property name="text">
    <string>Save Debugging State</string>
   </property>
   <property name="toolTip">
    <string>Save state with debugging information (advanced)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionMacros">
   <property name="text">
    <string>&amp;Macros</string>
   </property>
  </action>
  <action name="actionPassiveAcquisition">
   <property name="text">
    <string>&amp;Passive Acquisition</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="text">
    <string>&amp;Save Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveScreenshot">
   <property name="text">
    <string>Export Screenshot...</string>
   </property>
  </action>
  <action name="actionSaveMovie">
   <property name="text">
    <string>Export Movie...</string>
   </property>
  </action>
  <action name="actionSaveState">
   <property name="text">
    <string>Save State</string>
   </property>
  </action>
  <action name="actionLoadState">
   <property name="text">
    <string>&amp;Load State</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset application</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSaveWeb">
   <property name="text">
    <string>Export to Web...</string>
   </property>
   <property name="toolTip">
    <string>Export data to Web</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tomviz::CentralWidget</class>
   <extends>QWidget</extends>
   <header>CentralWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqCameraToolbar</class>
   <extends>QToolBar</extends>
   <header>pqCameraToolbar.h</header>
  </customwidget>
  <customwidget>
   <class>pqAxesToolbar</class>
   <extends>QToolBar</extends>
   <header>pqAxesToolbar.h</header>
  </customwidget>
  <customwidget>
   <class>pqPVAnimationWidget</class>
   <extends>QWidget</extends>
   <header>pqPVAnimationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqVCRToolbar</class>
   <extends>QToolBar</extends>
   <header>pqVCRToolbar.h</header>
  </customwidget>
  <customwidget>
   <class>tomviz::PipelineView</class>
   <extends>QTreeView</extends>
   <header>PipelineView.h</header>
  </customwidget>
  <customwidget>
   <class>tomviz::DataPropertiesPanel</class>
   <extends>QWidget</extends>
   <header>DataPropertiesPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tomviz::ModulePropertiesPanel</class>
   <extends>QWidget</extends>
   <header>ModulePropertiesPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tomviz::OperatorPropertiesPanel</class>
   <extends>QWidget</extends>
   <header>OperatorPropertiesPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqOutputWidget</class>
   <extends>QWidget</extends>
   <header>pqOutputWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqPythonShell</class>
   <extends>QWidget</extends>
   <header>pqPythonShell.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqLightsInspector</class>
   <extends>QWidget</extends>
   <header>pqLightsInspector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ParaView3/ParaView/Qt/Components/Resources/pqComponents.qrc"/>
  <include location="../ParaView3/ParaView/Qt/Core/Resources/pqCore.qrc"/>
  <include location="../ParaView3/ParaView/Qt/Widgets/Resources/QtWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
